examples/CURL/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-curl)
+
+hunter_add_package(CURL)
+find_package(CURL REQUIRED)
+
+
+
+message("CURL_INCLUDE_DIRS: ${CURL_INCLUDE_DIRS}")
+message("CURL_LIBRARIES: ${CURL_LIBRARIES}")
+message("CURL_FOUND: ${CURL_FOUND}")
+message("CURL_VERSION_STRING: ${CURL_VERSION_STRING}")
```

Comment:
This variables will not be defined if `find_package(CURL CONFIG REQUIRED)` used. Small overview:
* include directories and path to library stored in imported target CURL::libcurl
* CURL_FOUND not needed since we use REQUIRED (will crash if not found)
* version can be set by CMake modules, see: https://github.com/forexample/package-example/blob/cf2ea1d6a209fb9eca2ab83fdd0ac15fe4d3e807/Foo/CMakeLists.txt#L86